src/api/components/components.ts

```diff
@@ -475,9 +475,22 @@ class ConsumeTrigger extends ProTrigger {
 			name:        'Consume',
 			description: 'Applies skill effects when a player consumes an item',
 			data:        [
-				new MaterialSelect()
-					.setTooltip('The type of item that the player has consumed.')
+				new DropdownSelect("Material", 'material', 
+				['Any','Apple','Baked Potato','Beetroot','Beetroot Soup','Bread','Carrot','Chorus Fruit','Cooked Chicken',
+				'Cooked Cod','Cooked Mutton','Cooked Porkchop','Cooked Rabbit','Cooked Salmon','Cookie','Dried Kelp',
+				'Enchanted Golden Apple','Golden Apple','Glow Berries','Golden Carrot','Honey Bottle','Melon Slice',
+				'Milk bucket','Mushroom Stew','Poisonous Potato','Potato','Potion','Pumpkin Pie','Rabbit Stew','Beef',
+				'Chicken','Cod','Mutton','Porkchop','Rabbit','Salmon','Rotten Flesh','Spider Eye','Steak','Suspicious Stew',
+				'Sweet Berries','Tropical Fish'], 'Any')
```

Comment:
My only concern with this is that this list will be difficult to maintain. Does the Spigot Material interface have an `isEdible` method on it at all? We might be able to add another data entry to our data generation for consumable items as opposed to just the entire material list itself.

Reply:
Okay, tomorrow I will add entry "consumable" to the version data after checking the changes from recent versions.